src/components/core/Accordion/Accordion.tsx

```diff
@@ -69,9 +69,10 @@ export const Accordion = ({
   ...rest
 }: AccordionProps) => (
   <AccordionRoot multiple {...rest}>
-    {items.map(({ title, body, isDisabled, ...rest }) => (
+    {items.map(({ title, body, isDisabled, ...rest }, index) => (
```

Comment:
Matching rest of codebase

```suggestion
    {items.map(({ title, body, isDisabled, ...rest }, idx) => (
```